web/modules/custom/do_base/do_base.libraries.yml

```
@@ -0,0 +1,5 @@
+highlight_js.gherkin:
+  js:
+    https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.9.0/languages/gherkin.min.js: { type: external, minified: true }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)